cmd/kube-scheduler/app/server.go

```diff
@@ -58,6 +59,18 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/profile"
 )
 
+// TODO: delete this check after insecure flags removed in v1.24
+func checkNonZeroInsecurePort(fs *pflag.FlagSet) error {
+	val, err := fs.GetInt("port")
```

Comment:
We need to add this flag. you removed it in https://github.com/kubernetes/kubernetes/pull/96345/files?file-filters%5B%5D=.go#diff-9536691f7796a7d3d84a30c3ce6c1dd9d93317f9de3c142b8c8ae0d0d61e192eL163

Reply:
actually just remove this. since the flag is not registered anymore,

Reply:
Done